scaladoc/src/dotty/tools/scaladoc/tasty/ClassLikeSupport.scala

```diff
@@ -249,7 +249,7 @@ trait ClassLikeSupport:
   extension (c: ClassDef)
     def extractMembers: Seq[Member] = {
       val inherited = c.getNonTrivialInheritedMemberTrees.collect {
-        case dd: DefDef if !dd.symbol.isClassConstructor && !(dd.symbol.isSuperBridgeMethod || dd.symbol.isDefaultHelperMethod) => dd
+        case dd: DefDef if !dd.symbol.isClassConstructor && !(dd.symbol.isSuperAccessor || dd.symbol.isDefaultHelperMethod) => dd
```

Comment:
Alternatively, we could make sure to set the Artifact flag (which doesn't do anything at runtime but is used to signal to javac that this method should not be called from user code) on super accesors, default getters, etc.